client/src/test/scala/org/http4s/client/middleware/RetrySpec.scala

```diff
@@ -21,24 +21,25 @@ class RetrySpec extends Http4sSpec with Tables {
   }
 
   val defaultClient = Client.fromHttpService(route)
+  val failClient = Client(Service.const(Task.fail(new Exception("boom"))), Task.now(()))
```

Comment:
Are we using the failClient anywhere that required it's removal?

Reply:
I think I started another block for more tests and moved it and then abandoned that block.